eng/Versions.props

```diff
@@ -90,6 +90,7 @@
     <MicrosoftDiaSymReaderNativePackageVersion>1.7.0</MicrosoftDiaSymReaderNativePackageVersion>
     <!-- Infrastructure and test-only. -->
     <MicrosoftSourceLinkVersion>1.0.0-beta2-18618-05</MicrosoftSourceLinkVersion>
+    <MNAVersion>3.0.0-preview8-28379-02</MNAVersion>
```

Comment:
@dagood i have to specify as it was being set to empty. This and setting the NetCoreAppMaximum fixes the issue.

Reply:
This can't be a specific value, it varies per build. You shouldn't have to set it, it's set here:

https://github.com/dotnet/core-setup/blob/9528017ba529995cc42697abc1e3c59258303192/src/test/PrepareTestAssets/PrepareTestAssets.proj#L39-L47